fbw-a380x/src/wasm/systems/a380_systems/src/fuel/fuel_quantity_management_system.rs

```diff
@@ -28,6 +31,31 @@ enum ModeSelect {
     _Transfer,
 }
 
+#[derive(Deserialize)]
+struct ZfwParams {
+    target_zfw_cg: Vec<u32>,
+    target_zfw: Vec<ZfwRange>,
+}
+
+#[derive(Deserialize)]
+struct ZfwRange {
+    start: u32,
+    end: u32,
+}
+
+#[serde_as]
+#[derive(Deserialize)]
+struct TrimTankTables {
+    #[serde_as(as = "Vec<HashMap<DisplayFromStr, _>>")]
+    tables: Vec<HashMap<u32, Vec<u32>>>,
```

Comment:
Use `FxHashMap` (https://github.com/flybywiresim/aircraft/blob/master/fbw-common/src/wasm/systems/guidelines.md#hashmap-and-hashset)